### ISSUE TITLE
Suggestion: Combine petal dusts in a mortar

### ISSUE THREAD

**Issue by Febilian** [elucent/Roots#39]
Hear me out here!

It'd be neat if you could combine petal dusts of different types that fit into a similar category - such as offensive, defensive, utility, etc - and have the effects of more than one on a single staff. I was honestly disappointed when I could put a petal dust in the mortar but could not combine it with anything, so. Just a thought.

If you're worried about balance issues and overpowered spells, maybe have the modifiers used average out for each stat, such as if two petal dusts that are combined each have +1 efficiency to begin with, divide the combined stat total with the number of petal dusts put together. In this case, two. So you'd still end up with +1 efficiency if both components had efficiency already, but if only one did, it would be reduced to +0.5 on the resulting petal dust.

Basically for each stat, the stat total of the combined dust would be divided by how many petal dusts were combined into the final one. This would discourage combining every single petal dust you own together, because you couldn't have very powerful modifiers unless you heavily invested in only one of the modifier stats and ignored the rest.

You could also reduce the base efficiency/number of uses per staff depending on how many different dusts were used, decreasing drastically if you load up a petal dust with every single compatible effect. Maybe add an endgame method to bypass/alleviate this somehow, but the limitation should be there for a good while at the very least.

I apologize for all of these different suggestions, by the way. I'm just excited about this mod!

**Comment by werty1124** [comment]
This sounds similar to the crystal staff which can hold up to 4 different effects.

**Comment by elucent** [comment]
This isn't a bad idea, but I will probably repurpose it. Early-game, the spells are all intentionally single-effect because having more than one effect would be pretty powerful. In combat, let's say you add a Blistering Cold component onto a Rose's Thorns component. Without adding any modifiers, that's six hearts of damage, more powerful than if you added a level of potency to the Thorns staff, and without being gated behind the nether.

I can imagine this being added late-game possibly, not made in a mortar but in a ritual similar to the crystal staff one, where you infuse an item with multiple effects and they all get cast together. I do think there's some fun combinations available with this mechanic, like making your Rose's Thorns ignite enemies, or making your block breaking spell get the Blistering Cold component so you can trap enemies in pits. 

So I'll consider this for the future, but I think even without thinking of modifiers at all, this would unbalance the early-game.

**Comment by Febilian** [comment]
I see what you mean. I appreciate you at least taking the time to read my suggestions, by the way, even if they aren't all ones that work with the vision you have for the mod. I'm happy if I can at least provide any inspiration/feedback in some way!
